Font.js

```diff
@@ -1,94 +1,9 @@
-// basic inmports
-import { manPage } from "./src/manpage.js";
```

Comment:
no one's going to use this, so I might as well remove it, neat as I personally think it is.

Reply:
:thu